[IMP] Setting default payment term from supllier

alessandro-fiorino opened the pull request:
Porting of [#1061 ](https://github.com/OCA/purchase-workflow/pull/1061) to 13.0

alessandro-fiorino committed:
Merge pull request #1 from OCA/13.0

Update 13.0

alessandro-fiorino committed:
Setting PO payment term from supplier default

Made black happy

alessandro-fiorino commented:
Can it be merged ?

HviorForgeFlow commented:
Can you rebase to drop https://github.com/OCA/purchase-workflow/pull/1145/commits/c6d8c689f726182c5f2174fce59b071d7adb81e1 and squash other commits?

Thanks!

HviorForgeFlow commented:
ping @alessandro-fiorino

alessandro-fiorino committed:
Merge branch 'OCA:13.0' into 13.0

alessandro-fiorino committed:
[IMP] Don't reset order type if not necessary

When changing order company, the order type should not be reset if it's ok also for the new company.
